canvas_modules/common-canvas/src/common-properties/controls/checkboxset/checkboxset.scss

```diff
@@ -22,6 +22,15 @@
 		margin-bottom: $spacing-03;
 	}
 	.properties-checkboxset-container {
+		.wrapper{
+			display: flex;
+			justify-content:space-between;
+			width: 90px;
```

Comment:
why 90px?  What happens when the editor get larger/smaller?

Reply:
Removed it. used Grid to align icons & checkbox